terraform/modules/gro_ingestion_service/lambdas/src/models/EventRecord.ts

```diff
@@ -43,3 +44,28 @@ export const mapToEventRecord = (image: { [key: string]: AttributeValue }): Even
         Postcode: image["Postcode"].S ?? "",
     }
 }
+
```

Comment:
for the one above here we can do `unmarshal(image) as EventRecord` as using `@aws-sdk/util-dynamodb`